static/js/protein/chart.ts

```diff
@@ -313,9 +418,9 @@ export function drawTissueScoreChart(id: string, data: ProteinStrData[]): void {
     .attr("y", (d) => y(d.value))
     .attr("height", (d) => height - y(d.value))
     .attr("width", x.bandwidth())
-    .style("fill", (d) => TISSUE_COLOR_DICT[d.name])
+    .style("fill", (d) => ORGAN_COLOR_DICT[TISSUE_ORGAN_DICT[d.name]])
```

Comment:
do we know that all names in reformattedData will be in TISSUE_ORGAN_DICT? maybe we can have a default color for cases where d.name is not in TISSUE_ORGAN_DICT or the organ that d.name is part of isn't in ORGAN_COLOR_DICT?